src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2ProtocolException.cs

```diff
@@ -16,6 +16,12 @@ public Http2ProtocolException(Http2ProtocolErrorCode protocolError)
             ProtocolError = protocolError;
         }
 
+        public Http2ProtocolException(string message)
+            : base(message)
```

Comment:
nit: The `: base(message)` can be on the line above with the rest of the constructor method.

This matches the pattern below. The constructor pattern above is on two lines because putting it on one line would exceed the generally accepted line limit. We break up long lines (generally around 120 characters) to improve readability in the source code.

Reply:
makes sense. updated.